python/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1
+
+ARG PYTHON_VERSION=3.11
+FROM python:${PYTHON_VERSION}-slim as base
+
+WORKDIR /magika
+
+RUN --mount=type=cache,target=/root/.cache/pip \
```

Comment:
Is the /root/.cache/pip location best-practice? /cc @invernizzi

Reply:
yep I understand the rationale, I'm just wondering whether the specific path is what's considered best-practice, vs. something like /var/cache/pip? is there a well-defined best practice to where to put the cache?